### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by sffamily**
for v2.0.0

### CODE REVIEW

**Comment by sffamily** [comment]
@zhenyasav somehow I cannot add additional reviewer (I'm not listed as collaborator), so ping you here. I'll ask codeowner to add you as reviewer

**Review by LongOddCode — APPROVED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,75 @@
-# Change Log
+# Changelog
 
-All notable changes to the "teamsfx-toolkit" extension will be documented in this file.
+## 2.0.0 - May 25 2021
+2.0.0 is a major version upgrade for Teams Toolkit with many new improvements and supports.
+- Add: new design of UI and command palette list
+- Add: new getting started guide, samples and doc link from toolkit
+- Add: new sign-in experience for M365 and Azure
+- Add: new interactive flow for screating new app
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:8]: should it be "creating"?

File: packages/vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,75 @@
...
+- Add: add message extension capability to Teams app
+- Add: sample codes for in-meeting app.
+- Add: add Typescript support
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:11]: Typescript was supported in 1.2

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,75 @@
...
+- Add: support e2e dev experience for tab app hosted by either Azure or SPFx
+- Add: support adding backend API (Azure Functions) and SQL DB to tab app
+- Add: remote debugging support for tab app, bot app, and message extension
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:14]: Let's not say "remote debugging", but use "preview"

Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:14]: should be "messaging extension"

File: packages/vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,75 @@
...
+# Changelog
 
-All notable changes to the "teamsfx-toolkit" extension will be documented in this file.
+## 2.0.0 - May 25 2021
+2.0.0 is a major version upgrade for Teams Toolkit with many new improvements and supports.
+- Add: new design of UI and command palette list
+- Add: new getting started guide, samples and doc link from toolkit
+- Add: new sign-in experience for M365 and Azure
+- Add: new interactive flow for screating new app
+- Add: add message extension capability to Teams app
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:9]: messaging

File: packages/vscode-extension/CHANGELOG.md [hunk 5]
```diff
@@ -1,9 +1,75 @@
...
+- Add: add Typescript support
+- Add: support e2e dev experience for tab app hosted by either Azure or SPFx
+- Add: support adding backend API (Azure Functions) and SQL DB to tab app
+- Add: remote debugging support for tab app, bot app, and message extension
+- Add: dev environment checking and auto-setup
+- Add: cloud provision and deploy for Teams app from treeview UI and command palette
+- Add: support simplified auth code and graph client through scaffolding
+- Add: Mac/Linux support of new toolkit
+- Add: integration with GitHub Codespaces for dev/test
+- Enhancement: improved helloworld app through scaffolding for tab app, bot app, and message extension
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/CHANGELOG.md:20]: messagin

**Review by zhenjiao-ms — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (sffamily) [reply packages/vscode-extension/CHANGELOG.md:8]: followed some other CHnagelog, but I can use the Docker extension format. "Added""Fixed""Enhanced", fixing now.

File: packages/vscode-extension/CHANGELOG.md (hunk 2 above)
Comment (sffamily) [reply packages/vscode-extension/CHANGELOG.md:11]: it was in our Wiki's release notes, yes i can see previous changelog has it, so I'll remove this from our v2 changelog.

File: packages/vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (sffamily) [reply packages/vscode-extension/CHANGELOG.md:14]: will fix. Long found an issue that cannot merge, so will fix all and create a new PR.

Comment (sffamily) [reply packages/vscode-extension/CHANGELOG.md:14]: good catch, will global replace.

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (zhenjiao-ms) [reply packages/vscode-extension/CHANGELOG.md:8]: No, I mean "screating" should be "creating"

Comment (sffamily) [reply packages/vscode-extension/CHANGELOG.md]: this has been fixed. and made some more formating

**Review by zhenjiao-ms — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request closed without merging**